py/modsys.c

```diff
@@ -49,6 +49,7 @@ extern struct _mp_dummy_t mp_sys_stderr_obj;
 
 #if MICROPY_PY_IO && MICROPY_PY_SYS_STDFILES
 const mp_print_t mp_sys_stdout_print = {&mp_sys_stdout_obj, mp_stream_write_adaptor};
+const mp_print_t mp_sys_stderr_print = {&mp_sys_stderr_obj, mp_stream_write_adaptor};
```

Comment:
I wouldn't do it like that. Why do we need this if we know that majority of ports don't have dedicated stderr stream and it defaults to same stream as stdout/plat? This would make a difference if we allowed overriding sys.stderr, but we don't, and it will take its own time and usecases before we will. Also, what about the way the unix port prints exceptions currently? There's one mp_stderr_print, you introduce another => mess.

I'd suggest to start low-profile, like we usually do in uPy: let MP_PYTHON_ERROR_PRINTER default to &mp_plat_print, like before. But a port can define it to something else, which the unix port will do.